lib/collector.js

```diff
@@ -9,7 +9,7 @@ class DataCollector {
     const self = this;
     if (typeof info !== 'object' || !info.opcode ) return;
 
-    if (info.opcode.name.includes("PUSH") && info.stack.length > 0){
+    if (info.opcode.name.includes("PUSH1") && info.stack.length > 0){
```

Comment:
Hope this ok - the VM is showing that PUSH1 and PUSH2 are called with the hash at the top of the stack, resulting in duplicate hits. This makes everything contingent on a very specific implementation detail at Solidity though...